api/v01/vrp.rb

```diff
@@ -227,7 +237,8 @@ class Vrp < APIBase
                       status: :killed,
                       avancement: job.message,
                       graph: solution[:graph]
-                    }
+                    },
+                    geojsons: solution[:result] ? [solution[:result]].flatten(1).first[:geojsons] : nil
```

Comment:
This would not work, we would rather need something like

```suggestion
                    geojsons: solution[:result]&.first&[:geojsons]
```

but this returns _false_ if there is no solution[:result], we want to return _nil_ in this case.

Reply:
we can add `|| nil` to force the false to become nil